beater/server.go

```diff
@@ -36,7 +43,9 @@ func newServer(config Config, publish successCallback) *http.Server {
 	}
 
 	mux.HandleFunc("/healthcheck", func(w http.ResponseWriter, r *http.Request) {
+		requestCounter.Inc()
```

Comment:
i'd remove  `healthcheck`counters, don't think they provide any value.

Reply:
I would like to keep this in to make sure `# of request = actual number of requests`. Otherwise if for whatever reason `healthcheck` is hammered and the server is slowing down but the request counter stays low it might imply the it's not a request problem.